magento/magento2
# Product list does not contain extension_attributes stock_item

### Description (*)
 This pull request (PR) will be merge the added stock item extension_attributes in product list rest API /rest/V1/products?searchCriteria

### Related Pull Requests
 1.I added observer for "catalog_product_collection_load_after" in cataloginventory module.

### Fixed Issues (if relevant)

1. Fixes #36533

### Manual testing scenarios (*)
1.Open the Postman

2.Add the API URL: {baseurl}/rest/V1/products?searchCriteria

3.Method: GET

4.Authorization "Use Admin Token"

5.Check in response There is a "stock_item": object added in extension_attributes section.
